docker-compose.yml

```diff
@@ -11,9 +11,9 @@ services:
       - ./backend:/app
       - ./backend/media:/vol/web/media
     command: >
-      sh -c "python manage.py wait_for_db &&
-             python manage.py migrate &&
-             python manage.py runserver 0.0.0.0:8080"
+      sh -c "python manage.py runserver 0.0.0.0:8080 &&
+             python manage.py wait_for_db &&
+             python manage.py migrate"
```

Comment:
Why did you change the order of commands?

Reply:
There was an issue while running _docker-compose up_ as the process stuck at "_Attaching to py-cinema-full-stack-app-1, py-cinema-full-stack-db-1_", that's why I moved commands. 
I returned it now to initial version and it still works. Probably, restart of docker helped to solve problem and not moving commands.